torchci/components/additionalTestInfo/TestInfo.tsx

```diff
@@ -0,0 +1,305 @@
+      </div>
+    </>
+  );
+}
+
+function TDInfo({
+  workflowId,
+  jobs,
+  runAttempt,
+}: {
+  workflowId: string;
+  jobs: JobData[];
+  runAttempt: string;
+}) {
+  const shouldShow =
+    jobs.every((job) => job.conclusion !== "pending") &&
+    jobs.some((job) => job.name!.includes("/ test "));
+  const { data: info, error } = useSWR(
+    shouldShow
+      ? `https://ossci-raw-job-status.s3.amazonaws.com/additional_info/td_exclusions/${workflowId}/${runAttempt}`
+      : null,
+    fetcher
+  );
+
+  if (!shouldShow) {
+    return <div>Workflow is still pending or there are no test jobs</div>;
+  }
+
+  if (error) {
+    return <div>Error retrieving data {`${error}`}</div>;
+  }
+
+  if (!info) {
+    return <div>Loading...</div>;
+  }
+
+  if (Object.keys(info).length == 0) {
+    return (
+      <div>No test files were excluded or there was trouble parsing data</div>
+    );
+  }
+  return (
+    <div>
+      <div
+        style={{ fontSize: "1.17em", fontWeight: "bold", paddingTop: "1em" }}
+      >
+        Excluded test files by job
+      </div>
+      <RecursiveDetailsSummary
+        info={info}
+        level={1}
+        summaryFunction={(name: any, info: any) => {
+          const count = Object.values(info).reduce(
+            (prev: number, curr: any) => curr.length + prev,
+            0
+          );
+          return (
+            <>
+              {name} ({count})
+            </>
+          );
+        }}
+      >
+        {(config: any, configInfo: any, numSiblings: number) => (
+          <details open={numSiblings == 1} key={config}>
+            <summary>
+              {config} ({configInfo.length})
+            </summary>
+            <div
+              style={{
+                overflowY: "auto",
+                maxHeight: "50vh",
+                borderColor: "black",
+                borderStyle: "solid",
+                borderWidth: "1px",
+                padding: "0em 0.5em",
+              }}
+            >
+              {Array.from(configInfo).map((file: any) => {
+                return <div key={file}>{file}</div>;
+              })}
+            </div>
+          </details>
+        )}
+      </RecursiveDetailsSummary>
+    </div>
+  );
+}
+
+export function TestInfo({
+  workflowId,
+  runAttempt,
+  jobs,
+}: {
+  workflowId: string;
+  runAttempt: string;
+  jobs: JobData[];
+}) {
+  const [showInfo, setShowInfo] = useState("TD Info");
+
+  function setShowInfoHelper(info: string) {
+    if (showInfo === info) {
+      setShowInfo("none");
+    } else {
+      setShowInfo(info);
+    }
+  }
+
+  function ButtonSelector({ name }: { name: string }) {
+    return (
+      <button
+        onClick={() => setShowInfoHelper(name)}
+        className={showInfo === name ? styles.active : ""}
+      >
+        {name}
+      </button>
+    );
+  }
+
+  return (
+    <>
+      <div className={styles.tab}>
+        <ButtonSelector name="TD Info" />
+        <ButtonSelector name="Reruns Info" />
```

Comment:
Consider swapping the order of these two. I suspect the Reruns info is more useful to most customers